packages/quizzes-dashboard/pages/courses/[id].tsx

```diff
@@ -31,9 +29,9 @@ const CourseTitleWrapper = styled.div`
   justify-content: space-between;
 `
 
-const ShowCoursePage = ({ router }: any) => {
-  const dispatch = useDispatch()
-  const id = router.query.id
+const ShowCoursePage = () => {
+  const router = useRouter()
+  const id: any = router.query.id
```

Comment:
const quizId = router.query.quizId.toString()

That way if this is an array it gets converted to a string and we don't need the any annotation. This is better than converting `string | string[]` to any because it transforms a type error to a not found error.